app/Filament/Terrain/Widgets/QuickActionsWidget.php

```
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Filament\Terrain\Widgets;
+
+use Filament\Widgets\Widget;
+use ToneGabes\Filament\Icons\Enums\Phosphor;
+
+class QuickActionsWidget extends Widget
+{
+    protected static ?int $sort = 0;
+
+    protected static string $view = 'filament.terrain.widgets.quick-actions';
```

PHPStan (2.2.8)

[error] Static property App\Filament\Terrain\Widgets\QuickActionsWidget::$view overrides non-static property Filament\Widgets\Widget::$view.


(property.static)